src/screens/Menu.js

```diff
@@ -423,6 +423,17 @@ const MenuPage = () => (
         <Property name="menu.item">
           <Description>Any valid Button props for the menu items.</Description>
           <PropertyValue type="object">
+            <Example>{`{ alignSelf: 'start' }`}</Example>
+          </PropertyValue>
+        </Property>
+
+        <Property name="menu.item.align">
+          <Description>
+            How to align the contents along the cross axis. Any 'align-items'
+            valid CSS value is accepted, including composed ones such 'first
+            baseline' and 'unsafe start'.
+          </Description>
+          <PropertyValue type="string">
             <Example>{`{ align: 'start' }`}</Example>
```

Comment:
```suggestion
            <Example>'start' </Example>
```
Can we just change this example to a string since we specify the align part in the property name

Reply:
agree